Client/src/Validation/validation.js

```
@@ -174,6 +174,43 @@ const imageValidation = joi.object({
+	timezone: joi.string().trim().messages({ "string.empty": "Timezone is required." }),
+	color: joi.string().trim().messages({ "string.empty": "Color is required." }),
+	theme: joi.string(),
+	monitors: joi.array().min(1).items(joi.string().required()).messages({
+		"string.pattern.base": "Must be a valid monitor ID",
+		"array.base": "Monitors must be an array",
+		"array.empty": "At least one monitor is required",
+		"any.required": "Monitors are required",
+	}),
+	logo: logoImageValidation,
+	showUptimePercentage: joi.boolean(),
+	showBarcode: joi.boolean(),
```

Biome (1.9.4)

[error] This property value named showBarcode is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named showBarcode


(lint/suspicious/noDuplicateObjectKeys)